kontrakter/politi/varetekt/1.0/begjaeringVaretekt.schema.json

```diff
@@ -1,44 +1,42 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "description": "Skjema for begjæring av varetektsmeldinger fra påtale til tingretten",
+  "$id": "politi/fengsling/1.0/begjaeringVaretekt",
```

Comment:
Fra  https://datatracker.ietf.org/doc/html/rfc3986 :
>The "$id" keyword defines a URI for the schema, and the base URI that
   other URI references within the schema are resolved against.  The
   "$id" keyword itself is resolved against the base URI that the object
   as a whole appears in.
   If present, the value for this keyword MUST be a string, and MUST
   represent a valid URI-reference [[RFC3986](https://datatracker.ietf.org/doc/html/rfc3986)].
   
Mener vi kan fjerne det da vi har hele schema i en fil. Nåværende er i hvert fall ikke gyldig.

Reply:
Ønsker å bruke $id til å unikt identifisere et Schema, f.eks: https://politiet.no/politi/fengsling/1.0/begjaeringVaretekt
politiet.no for alle politiet sine definisjoner (produsent).